Formula/wally.rb

```diff
@@ -0,0 +1,33 @@
+class Wally < Formula
+  desc "Modern package manager for Roblox projects inspired by Cargo"
+  homepage "https://github.com/UpliftGames/wally"
+  url "https://github.com/UpliftGames/wally/archive/refs/tags/v0.3.1.tar.gz"
+  sha256 "c9b8bcdbfc906848fac5a49d69f54d3b2b4830b23dda7cd47190c5624c2bf04c"
+  license "MPL-2.0"
+  head "https://github.com/UpliftGames/wally.git", branch: "main"
+
+  depends_on "pkg-config" => :build
+  depends_on "rust" => :build
+  depends_on "openssl@1.1"
```

Comment:
Can we use `openssl@3`?

Reply:
Can we throw an error in `install` when the version is newer than `0.3.1` as a reminder to check for OpenSSL 3 support? Something like this will do:

https://github.com/Homebrew/homebrew-core/blob/eb925279f6181ca92234288b4de2ef8effdd7615/Formula/gitoxide.rb#L37

Except the message should say something like `Please check for OpenSSL 3 support!`, with perhaps extra explanation in a comment next to it.